packages/tools/examples/ultrasoundAnnotation/index.ts

```diff
@@ -0,0 +1,277 @@
+element.oncontextmenu = (e) => e.preventDefault();
+
+element.id = 'cornerstone-element';
+element.style.width = '500px';
+element.style.height = '500px';
+
+content.appendChild(element);
+
+const instructions = document.createElement('p');
+instructions.innerText =
+  'Middle Click: Window Level\nRight Click: Zoom\n Mouse Wheel: Stack Scroll';
+
+content.append(instructions);
+// ============================= //
+let renderingEngine;
+const viewportId = 'US_STACK';
+
+addButtonToToolbar({
+  onClick: () => {
+    const viewport = renderingEngine.getViewport(viewportId);
+    viewport.scroll(1);
+  },
+  title: 'Scroll',
+});
+
+addButtonToToolbar({
+  onClick: () => {
+    const viewport = renderingEngine.getViewport(viewportId);
+    viewport.scroll(1);
+  },
+  title: 'Scroll',
+});
+
+addButtonToToolbar({
+  onClick: () => {
+    const toolGroup = ToolGroupManager.getToolGroup(toolGroupId);
+    const usAnnotation = toolGroup.getToolInstance(
+      UltrasoundAnnotationTool.toolName
+    );
+    usAnnotation.setActiveAnnotationType(
+      UltrasoundAnnotationTool.USAnnotationType.BLINE
+    );
+  },
+  title: 'Add B-Line annotation',
+});
+
+addButtonToToolbar({
+  onClick: () => {
+    const toolGroup = ToolGroupManager.getToolGroup(toolGroupId);
+    const usAnnotation = toolGroup.getToolInstance(
+      UltrasoundAnnotationTool.toolName
+    );
+    usAnnotation.setActiveAnnotationType(
+      UltrasoundAnnotationTool.USAnnotationType.PLEURA
+    );
+  },
+  title: 'Add Pleura annotation',
+});
+
+addButtonToToolbar({
+  onClick: () => {
+    const toolGroup = ToolGroupManager.getToolGroup(toolGroupId);
+    const usAnnotation = toolGroup.getToolInstance(
+      UltrasoundAnnotationTool.toolName
+    );
+    usAnnotation.deleteLastPleuraAnnotation();
+
+    // Get the stack viewport that was created
+    const viewport = <Types.IStackViewport>(
+      renderingEngine.getViewport(viewportId)
+    );
+    viewport.render();
+  },
+  title: 'Delete last Pleura annotation',
+});
+
+addButtonToToolbar({
+  onClick: () => {
+    const toolGroup = ToolGroupManager.getToolGroup(toolGroupId);
+    const usAnnotation = toolGroup.getToolInstance(
+      UltrasoundAnnotationTool.toolName
+    );
+    usAnnotation.deleteLastBLineAnnotation();
+
+    // Get the stack viewport that was created
+    const viewport = <Types.IStackViewport>(
+      renderingEngine.getViewport(viewportId)
+    );
+    viewport.render();
+  },
+  title: 'Delete last B-line annotation',
+});
+
+addButtonToToolbar({
+  onClick: () => {
+    const toolGroup = ToolGroupManager.getToolGroup(toolGroupId);
+    const usAnnotation = toolGroup.getToolInstance(
+      UltrasoundAnnotationTool.toolName
+    );
+    // Get the stack viewport that was created
+    const viewport = <Types.IStackViewport>(
+      renderingEngine.getViewport(viewportId)
+    );
+    const percentage = usAnnotation.calculateBLinePleuraPercentage(viewport);
```

Comment:
Could this be done automatically in the render loop? Then we could assign it to the `toolData` and just grab it from there, instead of making a function call.